Cargo.toml

```diff
@@ -43,6 +43,7 @@ tokio = "1"
 tower-http = { version = "0.6", features = ["fs", "trace"] }
 tracing = "0.1"
 vinum = { git = "https://github.com/vinesystems/vinum.git", tag = "1.0.3" }
+regex = "1.11.1"
```

Comment:
There is a guideline for using dependency version : "If a crate version specifies the minor version but it is not critical to use that specific version, remove the minor version and leave only the major version." If using 1.11.1 is not required, I would like to suggest using only the major version.

Reply:
I will make the following changes.
`regex = "1"`